src/main/java/fi/aalto/cs/intellij/common/MalformedCourseConfigurationFileException.java

```diff
@@ -0,0 +1,21 @@
+package fi.aalto.cs.intellij.common;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class MalformedCourseConfigurationFileException extends Exception {
```

Comment:
the count of exceptions is growing, do they deserve a separate folder (like /common/exceptions) or something?

Reply:
This would definitely not be a bad idea